src/main/java/net/neoforged/neoforge/client/ClientHooks.java

```diff
@@ -1088,4 +1088,13 @@ public static RecipeBookType[] getFilteredRecipeBookTypeValues() {
         }
         return RECIPE_BOOK_TYPES;
     }
+
+    private static final RandomSource OUTLINE_PASS_RANDOM = RandomSource.create();
+
+    public static boolean isInTranslucentBlockOutlinePass(Level level, BlockPos pos, BlockState state) {
+        BakedModel model = Minecraft.getInstance().getBlockRenderer().getBlockModel(state);
+        OUTLINE_PASS_RANDOM.setSeed(42);
+        ChunkRenderTypeSet renderTypes = model.getRenderTypes(state, OUTLINE_PASS_RANDOM, level.getModelData(pos));
+        return renderTypes.contains(RenderType.TRANSLUCENT) || renderTypes.contains(RenderType.TRIPWIRE);
```

Comment:
Tripwire doesn't sort on upload, iirc.

Reply:
Tripwire does set the corresponding flag (second boolean parameter): `public static final RenderType TRIPWIRE = create("tripwire", DefaultVertexFormat.BLOCK, VertexFormat.Mode.QUADS, 1536, true, true, tripwireState());`

Reply:
Hmm, interesting; I distinctly remember that not being the case in some situation. Good to know!